Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -411,7 +411,16 @@ class BrowserViewController: UIViewController {
             self.view.alpha = (profile.prefs.intForKey(IntroViewControllerSeenProfileKey) != nil) ? 1.0 : 0.0
         }
 
-        if activeCrashReporter?.previouslyCrashed ?? false {
+        // Only ask to restore tabs from a crash if we had non-home tabs or tabs with some kind of history in them
+        guard let tabsToRestore = tabManager.tabsToRestore() else { return }
+        var onlyNoHistoryTabs = true
```

Comment:
Actually, every is an extension you added for Sync: https://github.com/mozilla/firefox-ios/blob/master/Sync/Info.swift#L69. Could we move this to our Shared/Utils library?